src/assets/css/index.scss

```diff
@@ -1,34 +1,34 @@
 // best seen at 1500px or less
 
-html, body { height: 100%; }
+html,
 body {
-  background:radial-gradient(ellipse at center, rgba(255,254,234,1) 0%, rgba(255,254,234,1) 35%, #B7E8EB 100%);
-  overflow: hidden;
+  height: 100%;
 }
 
-.ocean { 
-  height: 20%;
-  width:100%;
-  position:absolute;
-  bottom:0;
-  left:0;
+.ocean {
+  height: 5%;
+  width: 100%;
+  position: fixed;
+  bottom: 0;
+  left: 0;
   background: #015871;
 }
 
-
 .wave {
-  background: url(https://s3-us-west-2.amazonaws.com/s.cdpn.io/85486/wave.svg) repeat-x; 
+  background: url(https://s3-us-west-2.amazonaws.com/s.cdpn.io/85486/wave.svg)
```

Comment:
We should download the image to our asset or public file and use it from there. And the ocean effect can be down with pure css tho